code/modules/mob/mob_helpers.dm

```diff
@@ -398,4 +398,44 @@ mob/proc/get_standard_bodytemperature()
 	if(client?.player_details)
 		client.player_details.logging[smessage_type] += timestamped_message
 
-	return ..()
\ No newline at end of file
+	return ..()
+
+
+/proc/notify_ghosts(var/message, var/ghost_sound = null, var/enter_link = null, var/atom/source = null, var/mutable_appearance/alert_overlay = null, var/action = NOTIFY_JUMP, flashwindow = TRUE, ignore_mapload = TRUE, ignore_key, header = null, notify_suiciders = TRUE, var/notify_volume = 100) //Easy notification of ghosts.
+	if(ignore_mapload && SSatoms.initialized != INITIALIZATION_INNEW_REGULAR)	//don't notify for objects created during a map load
+		return
+	for(var/i in GLOB.observer_list)
+		var/mob/dead/observer/O = i
+		if(!O.client)
+			continue
+		if(!notify_suiciders && (O in GLOB.suicided_mob_list))
```

Comment:
I would check if suicide is handled properly - shooting yourself in the mouth (the do-after act) is our version of the suicide and may not be included here.

Reply:
Removed as we don't implement this GLOB anway.